### PR TITLE
Document `DROP FUNCTION`

### LINKED ISSUE #920: Document `DROP FUNCTION`
### Related code PR

https://github.com/risingwavelabs/risingwave/pull/9561

### Which part(s) of the docs might be affected or should be updated? And how?

Add a command

### Reference

_No response_

### PR DESCRIPTION
**Pull request by CharlieSYH**
## Info
- **Description**: 
[ What's changed? ]

- **Preview**: 
[ Paste the preview link to the edited page here. Edit this item after you submit the PR and the preview is ready. ]

- **Related code PR**: 
https://github.com/risingwavelabs/risingwave/pull/9561

- **Related doc issue**: 
Resolves https://github.com/risingwavelabs/risingwave-docs/issues/920

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

**Review by wangrunji0408 — APPROVED** [review]
LGTM!

File: docs/sql/commands/sql-drop-function.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+<drawer SVG={svg} />
+
+| Parameter or clause           | Description                                           |
+|-------------------------------|-------------------------------------------------------|
+| *function_name*               | Name of the UDF you want to drop.           |
+| ( *argument_type* [ , ... ] ) | Optional: Argument types of the function.<br/>Specify the argument types when the name of the function you want to drop isn't unique within the schema. |
+
+## Usage
+
+A function can be dropped either by:
```
Comment (hengm3467) [inline docs/sql/commands/sql-drop-function.md]: You use "either by" but there is not "or by", so I propose changing the wording to be something like:

A function can be dropped using one of the following methods:

Comment (hengm3467) [reply docs/sql/commands/sql-drop-function.md]: - Full function signature:

```sql
DROP FUNCTION function_name ( argument_type [, ...] ); 
```

- Function name only, if it is unique within its schema:

```sql
DROP FUNCTION function_name;
```

Comment (hengm3467) [reply docs/sql/commands/sql-drop-function.md]: You can add a note about using "SHOW FUNCTIONS" to list all available functions to see if a function is unique.

**Review by hengm3467 — COMMENTED** [review]
Otherwise looks good to me.

**Comment by CharlieSYH** [comment]
Updated. Please review it again, @hengm3467. https://github.com/risingwavelabs/risingwave-docs/pull/969/files#diff-17c04a2dd070ce96d005bfe71090af5fb5012185ce0a4657262089e1ca3c3234R43

**Review by hengm3467 — APPROVED** [review]
looks good. Thanks! @CharlieSYH

**Pull request merged**